lib/media/manifest_filterer.js

```diff
@@ -174,7 +176,11 @@ shaka.media.ManifestFilterer = class {
       for (const stream of streams) {
         if (stream.keyIds.size) {
           for (const keyId of stream.keyIds) {
-            const keyStatus = keyStatusMap[isGlobalStatus ? '00' : keyId];
+            let keyStatus = keyStatusMap[isGlobalStatus ? '00' : keyId];
+            if (!isGlobalStatus && !keyStatus && keyId.length == 22) {
```

Comment:
Where do these base64 key IDs come from?  Why aren't they converted to hex at the source of that data?

Reply:
Reverted!